hyperspy/_components/expression.py

```diff
@@ -36,19 +45,28 @@ class docstring.
         expression: str
             Component function in SymPy text expression format. See the SymPy
             documentation for details. The only additional constraint is that
-            the variable must be `x`. Also, in `module` is "numexpr" the
+            the variable(s) must be `x` (for 1D components) or `x` and `y` for
+            2D components. Also, if `module` is "numexpr" the
             functions are limited to those that numexpr support. See its
```

Comment:
Would remove brackets here:

"the variable(s) must be `x` for 1D components; or `x` and `y` for  2D components."